test/lit/wasm-split/profile-guided.wast

```diff
@@ -0,0 +1,67 @@
+;; Instrument the binary
+
+;; RUN: wasm-split --instrument %s -o %t.instrumented.wasm
```

Comment:
what does such a line do without a check later? just verify it doesn't crash?

Reply:
That's correct, but it also sets up state that is used in the subsequent `RUN` lines. Specifically, it writes the file %t.instrumented.wasm. The "%t" is replaced with a unique filename at runtime.